_data/chains/eip155-11155420.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "OP Sepolia Testnet",
+  "chain": "ETH",
+  "rpc": ["https://sepolia.optimism.io"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Sepolia Ether",
+    "symbol": "ETH",
+    "decimals": 18
+  },
+  "infoURL": "https://optimism.io",
+  "shortName": "opsep",
+  "chainId": 11155420,
+  "networkId": 11155420,
+  "explorers": [
+    {
+      "name": "opscout",
+      "url": "https://optimism-sepolia.blockscout.com/",
```

Comment:
after checking the failure 
need to cut the / off 
for the explorer url 

<img width="1004" alt="Screenshot 2566-11-22 at 22 31 59" src="https://github.com/ethereum-lists/chains/assets/113798442/c84798be-8445-4641-97ee-656195c942df">